docs/api-reference/io-columns-ranking.md

```diff
@@ -1,12 +1,12 @@
 ### Input and Output Columns
-The input label data type must be [key](xref:Microsoft.ML.Data.KeyDataViewType)
+The input label data type must be [Key](xref:Microsoft.ML.Data.KeyDataViewType)
 type or <xref:System.Single>. The value of the label determines relevance, where
 higher values indicate higher relevance. If the label is a
-[key](xref:Microsoft.ML.Data.KeyDataViewType) type, then the key index is the
+[Key](xref:Microsoft.ML.Data.KeyDataViewType) type, then the key index is the
 relevance value, where the smallest index is the least relevant. If the label is a
 <xref:System.Single>, larger values indicate higher relevance. The feature
 column must be a known-sized vector of <xref:System.Single> and input row group
-column must be [key](xref:Microsoft.ML.Data.KeyDataViewType) type. This trainer
+column must be [Key](xref:Microsoft.ML.Data.KeyDataViewType) type. This trainer
```

Comment:
>column must be [Key](xref:Microsoft.ML.Data.KeyDataViewType) type.  [](start = 0, length = 67)

let's add a blank line here to separate input and output into two paragraphs:

column must be [Key](xref:Microsoft.ML.Data.KeyDataViewType) type. 
(blank line)
This trainer outputs ....